examples/minimal.py

```diff
@@ -1,9 +0,0 @@
-import taichi as ti
```

Comment:
This is actually a useful file. Let's move it to `misc`.

Reply:
But we already have minimal_timed.py?